packages/web-components/src/utils/date-utils.ts

```diff
@@ -148,12 +149,12 @@ export const days = {
  * Return true if the year is a leap year.
  * (Exported for testing purposes)
  */
-export function checkLeapYear(year: number) {
+export function checkLeapYear(year: number): boolean {
```

Comment:
I'd love to rename this to `isLeapYear` but that may be more trouble than it's worth for what's essentially a personal preference.